CHANGELOG.md

```diff
@@ -44,10 +44,18 @@ Master list of UniV3 forks:
 
 ### Breaking changes
 
+* On Ethereum mainnet, the address `0xbbbbbBB520d69a9775E85b458C58c648259FAD5F`
+  (Bebop) is now on the list of restricted targets for the `BASIC` action
+
 ### Non-breaking changes
 
 * Fix bug that was causing bad revert reason for expired `AllowanceHolder`
   trades.
+* Add `BEBOP` action on Ethereum mainnet for 3rd-party RFQ provider Bebop. This
```

Comment:
Question, is this change adding bebop action ONLY for ethereum, or it is adding it for all chains? (The comment says only for ethereum ... )

In solver integration it would make more sense to have it available for any chain, otherwise we have to duplicate the implementation BEBOPCall for mainnet, BASICCall for all the other chains.